amundsen_common/fixtures.py

```diff
@@ -0,0 +1,224 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import string
+from typing import Any, List, Optional
+
+from amundsen_common.models.table import (Application, Column,
+                                          ProgrammaticDescription, Stat, Table,
+                                          Tag)
+from amundsen_common.models.user import User
+
+
+class Fixtures:
```

Comment:
Is the fixtures only for testing purpose? if so, we should move it under the tests folder.

Reply:
Ah, yes, it's for testing! But it's intended to be useful in multiple repos to create test objects easily. I don't think the tests/ folder gets carried over cleanly when importing in other repos.

Maybe amundsen_common/tests/fixtures.py would be preferable?

Reply:
yeah, amundsen_common/tests/fixtures.py would be better.

Reply:
- [ ] amundsen_common/tests/fixtures.py